packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -15,7 +15,9 @@ const StyledScrollableNav = styled.div`
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     white-space: nowrap;
     overflow-x: scroll;
-    scroll-behavior: smooth;
+
+    /* Avoid using smooth scrolling as it causes accessbility issues */
```

Comment:
```suggestion
    /* Avoid using smooth scrolling as it causes accessibility issues */
```

Reply:
doh, I'm blaming my mac keyboard on this :)